tools/setup_helpers/generate_code.py

```diff
@@ -44,7 +44,9 @@ def all_generator_source():
     'torch/csrc/autograd/generated/VariableType-3.cpp',
     'torch/csrc/autograd/generated/VariableType-4.cpp',
     'torch/csrc/autograd/generated/VariableType.h',
-    'torch/csrc/jit/generated/register_aten_ops.cpp',
+    'torch/csrc/jit/generated/register_aten_ops-0.cpp',
+    'torch/csrc/jit/generated/register_aten_ops-1.cpp',
+    'torch/csrc/jit/generated/register_aten_ops-2.cpp',
```

Comment:
Uh Python has a great library called `glob`. Can we just include all of `torch/csrc/jit/generated/register_aten_ops-*.cpp`?